Rx/v2/src/rxcpp/operators/rx-zip.hpp

```diff
@@ -200,55 +241,120 @@ struct zip : public operator_base<rxu::value_type_t<zip_traits<Coordination, Sel
     }
```

Comment:
it has been :)

I haven't yet heard of an issue where having the members be public induced someone to muck with them and break the operators contract. That would push me over the edge to making the members private.

I do however consider the members to be implementation details that could change in minor releases. This means that the members should be private now.

Conclusion - I am not going to require the members to be private, but I now recommend that they are made private.